bgpd/bgp_evpn.c

```diff
@@ -6160,29 +6136,30 @@ void bgp_evpn_encode_prefix(struct stream *s, const struct prefix *p,
 			stream_put(s, label + 1, BGP_LABEL_BYTES);
 		break;
 
-	case BGP_EVPN_IMET_ROUTE:
```

Comment:
yet again - a whole block of changes for whitespace, just moving the location of the open-brace?